### PR TITLE
Fix `databricks_user` creation with `force` on account

### LINKED ISSUE #1549: [ISSUE] `databricks_user` with `force` on account does not always work
### Configuration
```hcl
resource "databricks_user" "uc_admin" {
  for_each  = toset(module.defaults.admins)
  user_name = each.key
  force     = true
}
```

### Expected Behavior
`terraform apply` goes on

### Actual Behavior
`terraform apply` tells that users are in the account already

### Terraform and provider versions
v1.0.1

### PR DESCRIPTION
**Pull request by nkvuong**
Backend API changes error messages, updating code to match

Closes #1549

### CODE REVIEW

File: scim/resource_user.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func createForceOverridesManuallyAddedUser(err error, d *schema.ResourceData, us
 	}
 	// corner-case for overriding manually provisioned users
 	userName := strings.ReplaceAll(u.UserName, "'", "")
-	force := fmt.Sprintf("User with username %s already exists.", userName)
+	force := fmt.Sprintf("User with email %s already exists in this account", userName)
```
Comment (alexott) [inline scim/resource_user.go:88]: Will it continue to work with the workspace level identities?

File: scim/resource_user_test.go [hunk 2]
```diff
@@ -459,7 +459,7 @@ func TestCreateForceOverwriteCannotListUsers(t *testing.T) {
 		d := ResourceUser().TestResourceData()
 		d.Set("force", true)
 		err := createForceOverridesManuallyAddedUser(
-			fmt.Errorf("User with username me@example.com already exists."),
+			fmt.Errorf("User with email me@example.com already exists in this account"),
```
Comment (alexott) [inline scim/resource_user_test.go:462]: Maybe move this format string into a constant, or create a function that will accept email & return string?

Comment (nkvuong) [reply scim/resource_user_test.go]: refactored the error messages into a function

File: scim/resource_user.go (hunk 1 above)
Comment (nkvuong) [reply scim/resource_user.go]: you are correct - updated the correct error message

**Review by alexott — APPROVED** [review]

**Review by nfx — APPROVED** [review]

**Pull request merged**